### ISSUE TITLE
Unknown Error: 'NoneType' object is not iterable 

### ISSUE THREAD

**Issue by revission** [Kometa-Team/Kometa#233]
Processing Trakt List: https://trakt.tv/users/hammers-lists/lists/attenborough-documentaries
Unknown Error: 'NoneType' object is not iterable 

Tried w/ the :develop branch on docker and this failed on no specified branch. Still errors. It is a shared TV/Movie list, if that makes a difference.

**Comment by meisnate12** [comment]
duplicate of #166 waiting on trakt.py dependency

**Comment by revission** [comment]
Oooh, thanks. I missed that one! Great thing you have here! :-D
